mpu.ino

```diff
@@ -41,8 +41,15 @@ void readMPU6050() {
   if (micros() < lastCalcedMPU6050) {  // try to handle micros' long overflow in a harmless way
     lastCalcedMPU6050 = micros() - 10000;
   }
-
-  pitch = COMPLEMENTARY_FILTER_CONSTANT * ((pitch - pitchOffset) + rotationDPS_X * (micros() - lastCalcedMPU6050) / 1000000.000) + (1 - COMPLEMENTARY_FILTER_CONSTANT) * (degrees(atan2(accelerationY, accelerationZ)) - pitchOffset);  // complementary filter combines gyro and accelerometer tilt data in a way that takes advantage of short term accuracy of the gyro and long term accuracy of the accelerometer
+  // complementary filter combines gyro and accelerometer tilt data in a way that takes advantage of short term accuracy of the gyro and long term accuracy of the accelerometer
+  pitch = COMPLEMENTARY_FILTER_CONSTANT * ((pitch) + rotationDPS_X * (micros() - lastCalcedMPU6050) / 1000000.000)  // add rotation rate as measured by the gyro to current pitch - valid in short term
+          + (1 - COMPLEMENTARY_FILTER_CONSTANT) * (degrees(atan2(accelerationY, accelerationZ)) - pitchOffset);   // in the long term drift towards the angle of gravity measured by the accelerometer
+  if (robotEnabled) {
+    if (!wasRobotEnabled) {
+      pitchOffset = (pitch + pitchOffset);
+    }
+    pitchOffset = (pitch + pitchOffset) * (1 - .999991) + pitchOffset * (.999991);
```

Comment:
I might have already asked this, but why is the `.999991` not a variable?

Reply:
I've changed the number you asked about earlier to a constant this is a new number. I can make this into a constant too.